Cargo.toml

```diff
@@ -18,6 +18,7 @@ features = ["arduino-leonardo"]
 [profile.dev]
 opt-level = "s"
 lto = true
+codegen-units = 1
```

Comment:
Let's put a comment here that references the [upstream issue](https://github.com/rust-lang/compiler-builtins/issues/523#issuecomment-1547339559)